src/bot/tmi.js

```diff
@@ -178,7 +178,7 @@ class TMI extends Core {
       this.client[type].chat.on('WHISPER', async (message) => {
         message.tags.username = this.getUsernameFromRaw(message._raw)
 
-        if (!(await isBot(message.tags.username)) || !message.isSelf) {
+        if (!(commons.isBot(message.tags.username)) || !message.isSelf) {
```

Comment:
You can change `!(commons.isBot(sender.username))` -> `!commons.isBot(sender.username)`

Reply:
Same applies to other changes